elastic/kibana
# [Index management] Enable semantic_text feature

This PR enables semantic_text feature by default. 

How to test: 

1. Spin up Serverless or stack instance locally
2. Navigate to index management-> index overview page-> index mappings
3. Click add new field 
4. Ensure semantic_text field type is shown in Field type form by default


Tested in ESS and Serverless locally 

### Checklist

Delete any items that are not applicable to this PR.
